applications/nrf_desktop/Kconfig.debug

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config DESKTOP_LOG
+	bool "nRF Desktop logging"
+	help
+	  The option enables support for the nRF Desktop Logging.
```

Comment:
I think it can still be beneficial to have the help defined even for the default configuration as it can help the user to learn about the configuration when they skim through the Kconfig.debug file.

Reply:
I will extend the help information for DESKTOP_LOG as you suggested

Reply:
Ok, then let's leave it